packages/admin-panel/src/editor/BulkEditButton.js

```diff
@@ -12,7 +12,10 @@ import { openBulkEditModal } from './actions';
 
 export const BulkEditButtonComponent = ({ dispatch, value: recordId, actionConfig, row }) => {
   return (
-    <IconButton onClick={() => dispatch(openBulkEditModal(actionConfig, recordId, row))}>
+    <IconButton
+      className="bulk-edit-button"
```

Comment:
Out of curiosity, why do we use `className` on `IconButton` as opposed to `id`?

Reply:
Good catch, that was a mistake. Thanks